feedback/locale/fi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,104 @@
+# Finnish translation for the Helsinki Feedback Form
+# Copyright (C) 2018
+# This file is distributed under the same license as the helfeedback package.
+# Ville Koivune <ville.koivunen@hel.fi>, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-01-24 23:11+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Ville Koivunen <ville.koivunen@hel.fi>\n"
+"Language-Team: Finnish <dev@hel.fi>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: models.py:16
+msgid "The user agent string of the user's browser"
+msgstr "Käyttäjän selaimen user agent"
+
+#: models.py:18
+msgid "Rating"
+msgstr "Arvostelu"
+
+#: models.py:19
+msgid "Body"
+msgstr "Palaute"
+
+#: models.py:20
+msgid "URL of page where feedback was sent from"
+msgstr "URL, jolta palaute lähetettiin"
+
+#: models.py:21 models.py:123 templates/feedback/feedback_form.html:34
+#: templates/feedback/feedback_form.html:35
+msgid "Email"
+msgstr "Sähköposti"
+
+#: models.py:55
+#, python-format
+msgid "Unsupported notifier type: %s"
+msgstr "Epätuettu ilmoitustyyppi: %s"
+
+#: models.py:87
+#, python-brace-format
+msgid "New feedback received for {url}"
+msgstr "Uusi palaute {url}:ia koskien tullut"
+
+#: models.py:125
+msgid "User agent"
+msgstr "Selain"
+
+#: templates/feedback/feedback_form.html:3
+#: templates/feedback/feedback_form.html:6
+msgid "Give Us Feedback"
+msgstr "Anna palautetta palvelusta"
+
+#: templates/feedback/feedback_form.html:11
+msgid "How would you rate this service experience?"
+msgstr "Kuinka hyvin palvelu vastasi tarpeitasi?"
+
+#: templates/feedback/feedback_form.html:19
+msgid "Rate us on the scale of 1 (weak) - 4 (good)"
+msgstr "Arvostele asteikolla 1 (heikko) - 4 (hyvä)"
+
+#: templates/feedback/feedback_form.html:23
+msgid "How would you have us improve?"
+msgstr "Mitä voisimme vielä kehittää?"
+
+#: templates/feedback/feedback_form.html:24
+msgid "Please fill this field"
+msgstr "Täytä myös tämä kenttä"
+
+#: templates/feedback/feedback_form.html:28
+msgid ""
+"Should you wish for us to contact you, please leave your contact information"
+msgstr ""
+"Voit jättää yhteystietosi, jos haluat että otamme yhteyttä koskien "
+"palautettasi"
+
+#: templates/feedback/feedback_form.html:30
+#: templates/feedback/feedback_form.html:31
+msgid "Name"
+msgstr "Nimi"
+
+#: templates/feedback/feedback_form.html:35
+msgid "Please fill in an email address"
+msgstr "Sähköpostiosoite on väärin."
+
+#: templates/feedback/feedback_form.html:39
+msgid "Send"
+msgstr "Lähetä"
+
+#: templates/feedback/feedback_thankyou.html:3
+#: templates/feedback/feedback_thankyou.html:6
+msgid "Thank you!"
+msgstr "Kiitos palautteestasi!"
+
+#: templates/minimal.html:8
+msgid "Helsinki feedback"
+msgstr "Helsinki&#45;palaute"
```

Comment:
Why use the HTML character code here?

Reply:
Ah. Seems to have been a searching mishap. I though I should use a hyphen here (which is hard to type), but dash &mdash; (—) appears to be the correct one to use.

Reply:
Wait. I was thinking of the wrong place. Hyphen is actually the correct character here. The number is the only way to reliably input it.

Reply:
And given some sensibility training I've decided to repent my ways and use the normal ascii minus sign.